### PR TITLE
Fixes #1301

### LINKED ISSUE #1301: Change Toast to snackbars in app.
**Summary:** 

Snack bars look better than toast and disappear when activity is destroyed. 
https://stackoverflow.com/questions/34432339/android-snackbar-vs-toast-usage-and-difference

**Steps to reproduce:** 

Add a utils function in Utils.java

>     public static void showSnackbar(View view, String message) {
>         Snackbar.make(view, message, Snackbar.LENGTH_SHORT).show();
>     }

 **Commons app version:** 

2.6.7-Debug

**Screen-shots:** 

screenshot_20180313-202042

**Would you like to work on the issue**
Yes

### PR DESCRIPTION
**Pull request by ujjwalagrawal17**
## Description

Fixes #1301 

1. Add functions to show snabar in ViewUtil.java
2. Changed toast to Snackbar in NearbyAcitvity

## Tests performed

Manual Testing on Moto G5S+. (Debug Version)

## Screenshots showing what changed
 
screenshot_20180314-144440

### CODE REVIEW

File: app/src/main/java/fr/free/nrw/commons/utils/ViewUtil.java [hunk 1]
```diff
@@ -2,11 +2,22 @@
...
+import android.view.View;
 import android.widget.Toast;
 
 public class ViewUtil {
 
     public static void showLongToast(final Context context, @StringRes final int stringResId) {
         ExecutorUtils.uiExecutor().execute(() -> Toast.makeText(context, context.getString(stringResId), Toast.LENGTH_LONG).show());
     }
+
+    public static void showLongSnackbar(View view, String message) {
```
Comment (neslihanturan) [inline app/src/main/java/fr/free/nrw/commons/utils/ViewUtil.java]: This method is never used, can we remove? @ujjwalagrawal17

File: app/src/main/java/fr/free/nrw/commons/utils/ViewUtil.java [hunk 2]
```diff
@@ -2,11 +2,22 @@
...
 import android.content.Context;
 import android.support.annotation.StringRes;
+import android.support.design.widget.Snackbar;
+import android.view.View;
 import android.widget.Toast;
 
 public class ViewUtil {
 
     public static void showLongToast(final Context context, @StringRes final int stringResId) {
         ExecutorUtils.uiExecutor().execute(() -> Toast.makeText(context, context.getString(stringResId), Toast.LENGTH_LONG).show());
```
Comment (neslihanturan) [inline app/src/main/java/fr/free/nrw/commons/utils/ViewUtil.java]: This method is also never used, can we remove? @ujjwalagrawal17

File: app/src/main/java/fr/free/nrw/commons/nearby/NearbyActivity.java [hunk 3]
```diff
@@ -302,8 +302,7 @@ private void populatePlaces(List<Place> placeList) {
 
         if (placeList.size() == 0) {
             int duration = Toast.LENGTH_SHORT;
-            Toast toast = Toast.makeText(this, R.string.no_nearby, duration);
-            toast.show();
+            ViewUtil.showSnackbar(findViewById(R.id.container), R.string.no_nearby);
```
Comment (neslihanturan) [inline app/src/main/java/fr/free/nrw/commons/nearby/NearbyActivity.java]: duration variable is never used, if you remove that codacy will be happy @ujjwalagrawal17 :)

**Comment by ujjwalagrawal17** [comment]
@neslihanturan I had updated the code. Please tell me if any other improvements are needed.

**Comment by neslihanturan** [comment]
Thanks it seems good @ujjwalagrawal17 :)

**Pull request merged**